src/utils/utils.ts

```diff
@@ -1,5 +1,13 @@
 import { MetaTransaction } from 'ethers-multisend'
 
+import { DefaultSponsoredFunctions } from '../wallet/types'
+
+export const isDefaultSponsorisedFunction = (
```

Comment:
Should had condition on to. addOwner, to should ne wallet. setNonce to should be delays contract

Reply:
done